src/main/java/be/seeseemelk/mockbukkit/ServerMock.java

```diff
@@ -1934,18 +1935,18 @@ public boolean suggestPlayerNamesWhenNullTabCompletions()
 		throw new UnimplementedOperationException();
 	}
 
+	private static final Component noPermission = Component.text("I'm sorry, but you do not have permission to perform this command. Please contact the server administrators if you believe that this is in error.", NamedTextColor.RED);
+
 	@Override
 	public @NotNull String getPermissionMessage()
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		return net.kyori.adventure.text.serializer.legacy.LegacyComponentSerializer.legacyAmpersand().serialize(noPermission);
```

Comment:
You can use the Instance from MockUnsafeValues

Reply:
or at least use an import :P